redbot/cogs/mod/kickban.py

```diff
@@ -495,6 +495,57 @@ def remove_processed(ids):
                 await ctx.send(e)
             await ctx.send(_("Done. Enough chaos."))
 
+    @commands.command()
+    @commands.guild_only()
+    @commands.bot_has_permissions(move_members=True)
+    @commands.mod_or_permissions(move_members=True)
+    async def voicekick(
+        self, ctx: commands.Context, member: discord.Member, *, reason: str = None
+    ):
+        """Kick a member from a voice channel."""
+        author = ctx.author
+        guild = ctx.guild
+
+        if member.voice is None:
+            await ctx.send(_("{} is not in a voice channel.").format(member.name))
+            return
+        elif not await is_allowed_by_hierarchy(self.bot, self.settings, guild, author, member):
+            await ctx.send(
+                _(
+                    "I cannot let you do that. You are "
+                    "not higher than the user in the role "
+                    "hierarchy."
+                )
+            )
+            return
+        case_channel = member.voice.channel
+        # Store this channel for the case channel.
+
+        try:
+            await member.move_to(discord.Object(id=None))
+            # Work around till we get D.py 1.1.0, whereby we can directly do None.
+        except discord.HTTPException:
+            await ctx.send(_("Something went wrong while attempting to kick that member"))
+            return
+        except discord.Forbidden:  # Very unlikely that this will ever occur
```

Comment:
Your comment is almost correct. This will never occur. You need this catch above the catch of `discord.HTTPException`